src/resources/spanProcessor.ts

```diff
@@ -0,0 +1,32 @@
+import { BatchSpanProcessor, ReadableSpan } from '@opentelemetry/sdk-trace-base';
+import { logger } from '../logging';
+
+export class LumigoSpanProcessor extends BatchSpanProcessor {
+  override onEnd(span: ReadableSpan) {
+    if (shouldSkipSpanExport(span)) {
+      logger.debug('Not exporting span because it has NO_EXPORT=true attribute');
```

Comment:
hints should be in uppercase, actual value treated as case insensitive

Reply:
here the `true` value is a boolean value, & because we are in node I wrote it in lowercase.
This isn't the environment variable that the user sets, but an internal span attribute that we use